libqtile/sh.py

```diff
@@ -344,7 +343,4 @@ def loop(self) -> None:
                 continue
 
             val = self.process_line(line)
-            if isinstance(val, str):
-                print(val)
-            elif val:
-                pprint.pprint(val)
```

Comment:
We lose json pretty printing with this, right?

Reply:
Or, well, python object pretty printing.

Reply:
Right, that might be the case, this is something that was flagged as dead because of the `-> str` annotation on `process_line`. But, we can relax the `process_line` type annotation if we think we can get Python objects returned here.

Reply:
I suspect it's bogus, I'm getting pretty printing from somewhere at least,

```
screen.layout > info()
{'clients': ['Mozilla Firefox', 'urxvt', 'urxvt'],
 'current_stack': 1,
 'group': 'www',
 'name': 'stack',
 'stacks': [{'clients': ['Mozilla Firefox'], 'current': 0, 'split': False},
            {'clients': ['urxvt', 'urxvt'], 'current': 0, 'split': False}]}
```

Reply:
That's good to know. I relaxed the type on `process_line` to return `Any` so the mypy test doesn't complain here, and added the pprint back in.